package.json

```diff
@@ -11,9 +11,9 @@
     "deploy": "docusaurus deploy"
   },
   "dependencies": {
-    "by-node-env": "^2.0.1",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.56",
     "@docusaurus/core": "^2.0.0-alpha.56",
+    "@docusaurus/preset-classic": "^2.0.0-alpha.56",
+    "by-node-env": "^2.0.1",
```

Comment:
Looks like you built with `npm` not `yarn`. Could you revert this change from the pull request please and delete the `package-lock.json` file. Thanks